Base/usr/share/man/man7/sys.md

```diff
@@ -37,6 +37,48 @@ A file called `power_state` is responsible for power state switching.
 This file only responds to write requests on it. A written value of `1` results
 in system reboot. A written value of `2` results in system shutdown.
 
+### `kernel` directory
+
+This directory include two subdirectories - `net` and `variables`.
```

Comment:
(a series of short typo comments:)

includes

Reply:
I will fix whatever is present in the change, and keep the rest for another coming-soon PR :)